snowpack/src/build/file-urls.ts

```diff
@@ -0,0 +1,55 @@
+import path from 'path';
+import {SnowpackConfig} from '../types/snowpack';
+import {getExt, replaceExt} from '../util';
+
+export const defaultFileExtensionMapping = {
+  '.mjs': '.js',
+  '.jsx': '.js',
+  '.ts': '.js',
+  '.tsx': '.js',
+  '.vue': '.js',
+  '.svelte': '.js',
+  '.mdx': '.js',
+  '.svx': '.js',
+  '.elm': '.js',
+  '.yaml': '.json',
+  '.toml': '.json',
+  '.php': '.html',
+  '.md': '.html',
+  '.ejs': '.html',
+  '.njk': '.html',
+  '.scss': '.css',
+  '.sass': '.css',
+  '.less': '.css',
+};
+
+export function transformUrlForFile(
```

Comment:
could you add a comment describing what this method is doing exactly?

Also wondering: why not accepting an object with `{fileLoc, dirDisk, toUrl, config}` properties, instead of `(string, [string, string], string)`?

Reply:
👍 will add docs.

The intention was to match the interface of `getUrlForFile()`, but just with an added argument. Unlike `getUrlForFile()` this function is internal to our codebase, so with TypeScript I don't feel too strongly between the two forms. But I know @drwpow prefers the form that you're suggesting, so I'm happy to go that route.